osu.Game.Rulesets.Catch/UI/Catcher.cs

```diff
@@ -124,7 +134,11 @@ public Catcher([NotNull] Container trailsTarget, [NotNull] Container droppedObje
             InternalChildren = new Drawable[]
             {
                 hitExplosionPool = new DrawablePool<HitExplosion>(10),
-                caughtFruitContainer = new Container<DrawablePalpableCatchHitObject>
+                caughtFruitPool = new DrawablePool<CaughtFruit>(50),
```

Comment:
As a safety to avoid asynchronous loading, we'd usually put these inside the `[BackgroundDependencyLoader]` method, but I've checked this case is fine here (async happens from `CreateDrawableRuleset` and includes all `ctor`s).